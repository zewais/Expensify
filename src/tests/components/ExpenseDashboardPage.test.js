import React from "react"
import {shallow} from "enzyme"
import ExpesneDashboardPage from "../../components/ExpenseDashboardPage"

test("Should render the expense dashboard page", () => {
    const wrapper = shallow(<ExpesneDashboardPage/>)
    expect(wrapper).toMatchSnapshot();
})