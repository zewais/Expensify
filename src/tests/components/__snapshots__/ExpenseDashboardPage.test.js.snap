// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the expense dashboard page 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFIlters) />
  <Connect(ExpenseList) />
</div>
`;
